src/app/issue/settings/settings.component.ts

```diff
@@ -348,8 +356,14 @@ export class SettingsComponent implements OnInit {
    *
    */
   validDate(): boolean {
-    if (this.issue.issueDetail.startDate != undefined && this.issue.issueDetail.endDate != undefined) {
-      if (new Date(this.issue.issueDetail.startDate) >= new Date(this.issue.issueDetail.endDate)) {
+    if (
+      this.issue.issueDetail.startDate != undefined &&
+      this.issue.issueDetail.endDate != undefined
+    ) {
+      if (
+        new Date(this.issue.issueDetail.startDate) >=
+        new Date(this.issue.issueDetail.endDate)
```

Comment:
TIL: man kann Dates mit ">=" vergleichen, cool 👍 

Wenn man JS Müll machen möchte kann man die Funktion auch so zusammenfassen:
```js 
validateDate(): boolean {
  return (!this.issue.issueDetail.startDate && !this.issue.issueDetail.endDate) 
	? (this.issue.issueDetail.startDate >= this.issue.issueDetail.endDate)
	: true;
}
```

Reply:
Zusammenfassung kommt in den nächsten Minuten